crates/subspace-farmer/src/cluster/cache.rs

```diff
@@ -361,99 +361,34 @@ where
     caches_details
         .iter()
         .map(|cache_details| async move {
-            // Initialize with pending future so it never ends
-            let mut processing =
-                FuturesUnordered::<Pin<Box<dyn Future<Output = ()> + Send>>>::from_iter([
-                    Box::pin(pending()) as Pin<Box<_>>,
-                ]);
-            let subscription = nats_client
-                .queue_subscribe(
-                    ClusterCacheWritePieceRequest::SUBJECT
-                        .replace('*', &cache_details.cache_id_string),
-                    cache_details.cache_id_string.clone(),
+            let process = |ClusterCacheWritePieceRequest {
```

Comment:
Let's inline these `process` functions, I bet that way the diff (when whitespaces are ignored) will be much nicer since the order of old and new code will match rather than being inversed.

Reply:
fixed